guides/managing-sessions.rst

```diff
@@ -43,7 +43,7 @@ one single container object:
 
 .. note::
 
-    Mink will even lazy-start your sessions when needed (on first ``getSession()``
+    Mink will even lazy-start your sessions when needed (on the first ``getSession()``
```

Comment:
"real" doc content updates can easily be done in separate PRs. I have not attempted to check the truth of the statements, just fixed up some grammar.